src/Native/GirTestLib/girtest-typed-record-tester.c

```diff
@@ -224,6 +224,18 @@ int girtest_typed_record_tester_get_ref_count_sum_nullable(GirTestTypedRecordTes
     return girtest_typed_record_tester_get_ref_count_sum(data, size);
 }
 
+/**
+ * girtest_typed_record_tester_out_no_ownership_transfer:
+ * @data: (out) (transfer none): a `GirTestTypedRecordTester`
```

Comment:
A litte more explicit:

```suggestion
 * @data: (not nullable) (out caller-allocates) (transfer none): a `GirTestTypedRecordTester`
```